Sources/StreamVideo/Call.swift

```diff
@@ -239,14 +241,19 @@ public class Call: @unchecked Sendable, WSEventsSubscriber {
         }
         
         var settingsOverride: CallSettingsRequest?
+        var limits: LimitsSettingsRequest?
         if maxDuration != nil || maxParticipants != nil {
-            settingsOverride = CallSettingsRequest(
-                limits: .init(
-                    maxDurationSeconds: maxDuration,
-                    maxParticipants: maxParticipants
-                )
+            limits = .init(
+                maxDurationSeconds: maxDuration,
+                maxParticipants: maxParticipants
             )
         }
+
+        settingsOverride = CallSettingsRequest(
```

Comment:
Shall we add a CallCRUD test for this case? I'm thinking something with 2 users, one creating the call, sleep until the backstage starts and then try to join user 2. Now revert the case and try to join from user 2 before backstage start

Reply:
good point, added one